data/schema.graphql

```diff
@@ -1,11 +1,14 @@
+# source: https://metaphysics-staging.artsy.net/
+# timestamp: Tue Jun 26 2018 15:21:08 GMT-0400 (EDT)
+
 input AddAssetToConsignmentSubmissionInput {
-  # The type of the asset
+  """The type of the asset"""
```

Comment:
`"""` is so ugly. :-/

Reply:
Did `yarn relay` work for you? (I guess it would if you were able to push?) 

I tried running this locally and am getting syntax errors because of this change. Looks like due to: https://github.com/graphql-cli/graphql-cli/issues/199 and sounds like we have to update relay to support this new comment-style.

Reply:
That's a blocker for this PR today, we can't do that without serious work yet I'm afraid